app/models/profile.py

```diff
@@ -388,7 +388,7 @@ def validate_graph(self, data_graph: Graph) -> bool:
         shacl_graph.parse(str(self.shacl_sip()), format="turtle")
         shacl_graph.parse(str(self.shacl_profile()), format="turtle")
         conforms, results_graph, results_text = shacl_validate(
-            data_graph=data_graph, shacl_graph=shacl_graph, meta_shacl=True
+            data_graph=data_graph, shacl_graph=shacl_graph, meta_shacl=True, allow_warnings=True
```

Comment:
Deze heb ik aangepast omdat ik de SHACL nu opsplits met Warnings (wanneer er SHOULD staat in de SIP spec) en Violations (bij MUST).